linux_os/guide/system/bootloader-grub2/uefi/grub2_uefi_password/rule.yml

```diff
@@ -26,13 +26,11 @@ description: |-
     NOTE: the bootloader superuser account and password MUST differ from the
     root account and password.
     {{% endif %}}
+    {{% if "ubuntu" in product %}}
     Once the superuser password has been added,
     update the
     <tt>grub.cfg</tt> file by running:
-    {{% if "ubuntu" in product %}}
     <pre>update-grub</pre>
-    {{% else %}}
-    <pre>grub2-mkconfig -o {{{ grub2_uefi_boot_path }}}/grub.cfg</pre>
```

Comment:
Is the text change also valid to other products too? Like sle12 or wrlinux1019?
If not, instead of excluding the `else`, we should make it into a conditional that doesn't match rhel nor fedora.

Reply:
If I understand comment from @teacup-on-rockingchair correctly, configuration with `grub2-mkconfig` is valid for SLE.
So you should change the `else` to an `elif`.